packages/azure-kusto-data/package.json

```diff
@@ -29,26 +42,31 @@
         "directory": "azure-kusto-data"
     },
     "files": [
-        "source/**/*.js",
-        "source/**/*.d.ts",
-        "index.js",
-        "index.d.ts",
+        "dist/",
+        "dist-esm/src/",
+        "types/src/",
```

Comment:
Why aren't types under dist-esm? 
Since they are output, they should be under there and be git and npm ignored.

Reply:
Why do we need them locally?
They're generated from the typescript files, they shouldn't be anywhere except for our product.